patternfly/patternfly-elements
# fix: Deprecated function reference in pfe-modal

## pfe-modal: bug fix

pfe-modal was referencing `_mapSchemaToSlots` which was deprecated in the 1.0 release.

### Preview

- [pfe-modal](https://deploy-preview-1272--happy-galileo-ea79c4.netlify.app/elements/pfe-modal/demo/) 


### What has changed and why

Summarize files edited as part of this MR along with a brief description of what was changed/why.

- Removed the deprecated _mapSchemaToSlots reference from code and docs


### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. Open the [demo page](https://deploy-preview-1272--happy-galileo-ea79c4.netlify.app/elements/pfe-modal/demo/)
2. Open the first modal on the page, delete the p tag inside the modal.
3. Inspect the console and confirm no error messaging.

This is the error we see on the main branch:
```
pfe-modal.umd.js:111 Uncaught TypeError: _this._mapSchemaToSlots is not a function
    at MutationObserver.<anonymous> (pfe-modal.umd.js:111)
```

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
